docs/source/federation-2/backward-compatibility.mdx

```diff
@@ -130,6 +130,53 @@ type ProductDimensions {
 
 This `@requires` directive should _break_ composition because it depends on subfields of `ProductDimensions` that _don't exist_ (`length` and `depth`).
 
+#### Invalid `@provides` directives
+
+A subgraph can mark an entity field with the [`@provides` directive](https://www.apollographql.com/docs/federation/federated-types/federated-directives/#provides) to indicate that it resolve entity fields normally marked as `@external` if they are instead resolved through this particular path:
```

Comment:
I found this explanation a little difficult to follow. What does "it" refer to in, "that it resolve[s] entity fields normally..."? Could we callout whatever "it" is explicitly to clarify?

Reply:
Is it something like this:?

```suggestion
A subgraph can annotate an entity field with the [`@provides` directive](/federation/federated-types/federated-directives/#provides) to indicate that the subgraph can resolve entity fields normally marked as `@external` on its own.
```